src/api/app/models/user.rb

```diff
@@ -840,6 +841,11 @@ def mark_login!
     update_attributes(last_logged_in_at: Time.now, login_failure_count: 0)
   end
 
+  def send_metric_for_beta_change
+    channel = (in_beta? ? 'joined_beta' : 'left_beta')
+    RabbitmqBus.send_to_bus('metrics', "user.#{channel} state=#{state}")
```

Comment:
Why send `User.state`? I guess you just want to do is just `value=1`

Reply:
As [said](https://github.com/openSUSE/open-build-service/pull/5712#discussion_r212389218) I just picked a random field. I'll change it to value=1 then